spec/models/item_spec.rb

```diff
@@ -1,5 +1,34 @@
 require "rails_helper"
 
 RSpec.describe Item do
+  let(:valid_user_params) { { email: "bob@example.org", password: "secret" } }
```

Comment:
I'd just use it directly below, without defining `let` for things you don't use inside tests. Just a thought for future consideration.